dockerfiles/rust-1.43.Dockerfile

```diff
@@ -22,3 +23,6 @@ RUN rm -rf /app/src
 
 ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="Cargo.toml,Cargo.lock"
 
+
```

Comment:
@libmartinito what's the script you used to generate this? The spacing seems very inconsistent - in some files there are 2 empty lines between blocks, in some there are none - looks very haphazard